src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -216,10 +216,7 @@ public void InitializeSolutionBindingOnBackgroundThread()
         {
             this.solutionBindingOperation.RegisterKnownConfigFiles(this.InternalState.BindingConfigs);
 
-            var projectsToUpdate = GetProjectsForRulesetBinding(this.InternalState.IsFirstBinding, this.InternalState.BindingProjects.ToArray(),
-                this.host.Logger, this.threadHandling);
-
-            this.solutionBindingOperation.Initialize(projectsToUpdate);
+            this.solutionBindingOperation.Initialize();
```

Comment:
Previously `BindingProcessImpl` passed a list of projects to `SolutionBindingOperation`.

However, `SolutionBindingOperation` not longer uses the project list, so I removed the code from this class that calculated the set of projects.